### PR TITLE
Use cloud events envelope

### PR DESCRIPTION
**Pull request by Hyperkid123**
Part of: https://issues.redhat.com/browse/RHCLOUD-26019

### CODE REVIEW

File: rest/featureflags/featureflags.go [hunk 1]
```diff
@@ -18,6 +19,7 @@ type FFClient struct {
 var ffClient *FFClient
 
 func newClientWrapper(cfg *config.ChromeServiceConfig) (*unleash.Client, error) {
+	fmt.Println(cfg)
```
Comment (BlakeHolifield) [inline rest/featureflags/featureflags.go:22]: Do you still need this?

File: rest/cloudEvents/cloudEvents.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
+package cloudEvents
```
Comment (BlakeHolifield) [inline rest/cloudEvents/cloudEvents.go:1]: I think the standard is no mixed case for go, so `cloudevents` would be fine

**Review by BlakeHolifield — CHANGES_REQUESTED** [review]
Short feedback and small details

File: rest/featureflags/featureflags.go (hunk 1 above)
Comment (Hyperkid123) [reply rest/featureflags/featureflags.go:22]: Nope

**Review by BlakeHolifield — APPROVED** [review]

**Pull request merged**